CODEOWNERS

```diff
@@ -12,3 +12,29 @@
 # Observability
 /docs/observability.md       @npepinpe @ChrisKujawa
 /docs/assets/observability*  @npepinpe @ChrisKujawa
+
+# C8Run by Distro team
+.github/workflows/c8run* @camunda/distribution
+
+# E2E test suites by QA team
+.github/workflows/core-application-e2e-tests* @camunda/qa-engineering
+
+# General Automation, Unified CI and release helpers by Monorepo DevOps team
+/.github/workflows/add-to-projects.yml                   @camunda/monorepo-devops-team
+/.github/workflows/backport.yml                          @camunda/monorepo-devops-team
+/.github/workflows/camunda-helm-integration.yml          @camunda/monorepo-devops-team
+/.github/workflows/camunda-platform-release*             @camunda/monorepo-devops-team
+/.github/workflows/chatops*                              @camunda/monorepo-devops-team
+/.github/workflows/check-licenses.yml                    @camunda/monorepo-devops-team
+/.github/workflows/check-pr-conflicts.yml                @camunda/monorepo-devops-team
+/.github/workflows/ci*                                   @camunda/monorepo-devops-team
```

Comment:
There is currently only `ci.yml` but I plan to refactor larger jobs into separate `ci-...yml` files in the future.